### PR TITLE
Update all resources in RoslynDiagnosticAnalyzerResources

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx [hunk 1]
```diff
@@ -118,109 +118,109 @@
...
-    <value>Do not invoke Diagnostic.Descriptor</value>
+    <value>Do not invoke 'Diagnostic.Descriptor'</value>
   </data>
   <data name="DoNotInvokeDiagnosticDescriptorDescription" xml:space="preserve">
-    <value>Accessing the Descriptor property of Diagnostic in compiler layer leads to unnecessary string allocations for fields of the descriptor that are not utilized in command line compilation. Hence, you should avoid accessing the Descriptor of the compiler diagnostics here. Instead you should directly access these properties off the Diagnostic type.</value>
+    <value>Accessing the 'Descriptor' property of 'Diagnostic' in compiler layer leads to unnecessary string allocations for fields of the descriptor that are not utilized in command line compilation. Hence, you should avoid accessing the 'Descriptor' of the compiler diagnostics here. Instead you should directly access these properties off the 'Diagnostic' type.</value>
   </data>
   <data name="DoNotInvokeDiagnosticDescriptorMessage" xml:space="preserve">
     <value>Do not invoke property '{0}' on type '{1}', instead directly access the required member{2} on '{1}'</value>
```
Comment (mavasani) [inline src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx:172]:
```suggestion
    <value>Do not invoke property '{0}' on type '{1}', instead directly access the required member '{2}' on '{1}'</value>
```

Comment (Evangelink) [reply src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx:172]: We shouldn't change this (or at least not in this PR), the code is taking care of adding the space + wrapping in single quotes when needed: https://github.com/dotnet/roslyn-analyzers/blob/master/src/Roslyn.Diagnostics.Analyzers/Core/DoNotInvokeDiagnosticDescriptor.cs#L64

**Comment by Evangelink** [comment]
@mavasani I have replied to your comment, the resource is expected to be as it is currently. I can either keep it as-is, fix it and fix code here or create a ticket and do a separate PR to fix this.

File: src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx (hunk 1 above)
Comment (mavasani) [reply src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx:172]: Ah got you. Can you file a separate issue for fixing this up in a separate PR?

Comment (Evangelink) [reply src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx:172]: #3560

**Pull request merged**